toolkits/springer/packages/springer-hero/view/springer-hero.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="c-hero">
+	{{#if imageLeft}}
+		<div class="c-hero__image">
+			<img src="{{imageSrc}}" alt=""/>
```

Comment:
Would it be wise to add a variable for the `alt=""` too?

Reply:
I am going to consider it decorative (since it just illustrates the title, like "Astronomy"). Ideally, it _would_ have `alt` text anyway, but I'm worried about people using poor/placeholder `alt` text, which is worse than nothing. I see this a lot on the current site